utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -47,16 +47,6 @@ struct ThreadStats {
     classes: HashSet<&'static str>,
 }
 
-pub fn get_tid() -> usize {
-    let res = TID.with(|t| {
```

Comment:
Shouldn't this also remove the TID threadlocal?

Reply:
You are right.